mmv1/third_party/terraform/.teamcity/components/generated/settings.erb

```diff
@@ -10,7 +10,7 @@
 var defaultStartHour = 4
 
 // specifies the default level of parallelism per-service-package
-var defaultParallelism = 12
+var defaultParallelism = 20
```

Comment:
Parallelism set to 20, matching the current test project.

I have previously mentioned how this could need reducing once tests are split into multiple packages.